### ISSUE TITLE
Question: parallelize the apply command?

### ISSUE THREAD

**Issue by adimil** [soedinglab/MMseqs2#113]
Edit: sorry - found it on th Wiki page.

Is there a way to parallelize the apply command without manually splitting the DB?
Would MPI work in this module?

Thanks

**Comment by martin-steinegger** [comment]
Oh I am happy that you use `apply`. Please let us know if you have any problems.
Just for the record, if some other user have a similar problem. 
`Apply` can be used using MPI with the `RUNNER` variable but can also use all threads on single machine.
Information how to use `apply` can be found here https://github.com/soedinglab/mmseqs2/wiki#how-to-run-external-tools-for-each-database-entry

**Comment by adimil** [comment]
Hi Martin,
I use MMseqs2 Version: 8f44a3195116bd6932f559805a86b93b0e41da71-MPI
I ran a test on subdb of 5 clusters and it worked perfectly (without MPI).

I then tried to run on a large dataset with MPI on IBM Platform LSF, it seems like it is finished but did not merge back the files, is there a way to do it manually?

Also, what whould be the best way to check that it is really done? (I counted nulls and got the amount of clusters I used and looked at the last cluster printed out and it looks fine...)
